### DISCUSSION TITLE
Configurar Oracle com SID no Azure Data Studio

### DISCUSSION THREAD

**Discussion by rmunhozdev** [balta-io/forum#137] [Tira Dúvidas] [answered]
Como faço pra configurar o Data Source no Azure Data Studio com Oracle ( SID )?
Tentei por várias vezes e não consegui.

![image](https://user-images.githubusercontent.com/39678569/205112773-e4db2a04-27a5-4121-9c3b-7c485474b0a1.png)

**Comment by andrebaltieri** [comment, marked as answer]
@rmunhozdev precisa desta extensão:
https://learn.microsoft.com/en-us/sql/azure-data-studio/extensions/extension-for-oracle?view=sql-server-ver16

**Reply by rmunhozdev** [reply]
Opa @andrebaltieri , sim eu já havia instalado.
O detalhe é que tem o SID e no Azure não encontrei nada de como colocar no Data Source . No SQL Developer vai de boas . Olha só
![image](https://user-images.githubusercontent.com/39678569/205133264-831b9c40-e4b3-43fa-bba3-5c0ef9aa37b5.png)

Já no Azure eu tentei assim .. mas sem chances.
![image](https://user-images.githubusercontent.com/39678569/205133375-a1774aa3-0a12-4be1-be2c-233695073d28.png)

**Reply by rmunhozdev** [reply]
Ops acabei printando o modelo da extensão  rs.
Segue o do SQL Developer que funciona perfeitamente.
![image](https://user-images.githubusercontent.com/39678569/205133695-0c9201d5-4d1f-4cf9-ae5a-efb74fea261b.png)
